tests/jax/models/MNIST_CNN/test_mnistcnn.py

```diff
@@ -0,0 +1,89 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, Sequence
+
+import jax
+import jax.numpy as jnp
+import pytest
+import functools
+from flax import linen as nn
+from infra import ComparisonConfig, ModelTester, RunMode
+from transformers import AutoTokenizer, FlaxDistilBertForMaskedLM
+
+from CNN import CNN
+
+class MNISTCNNTester(ModelTester):
+    """Tester for MNIST CNN model."""
+
+    # @override
+    @staticmethod
+    def _get_model() -> nn.Module:
+        return CNN()
+
+    # @override
+    @staticmethod
+    def _get_forward_method_name() -> str:
+        return "apply"
+
+    # @override
+    @staticmethod
+    def _get_input_activations() -> Sequence[jax.Array]:
+        img = jnp.ones((28, 28, 1))
+        return img
+    
+    # @override
+    def _get_forward_method_args(self):
+        inp = self._get_input_activations()
+        init_model = self._model.init(jax.random.PRNGKey(42), inp, train=False)
+        return [init_model, inp]
+
+    # @override
+    def _get_forward_method_kwargs(self) -> Dict[str, jax.Array]:
+        return {"train": False}
+    
+    # @override
+    def _get_static_argnames(self):
+        return ['train']
```

Comment:
Also, this is a good example of having `train` as a kwarg and also a static arg used in `jax.jit`. Good job!